vagrant_variables.yml.openstack

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
It is possible.  But I figured that it was different enough that not having a bunch of conditional stuff would be cleaner.  We can always look into this in the future if things get unwieldy.